cmd/operator-sdk/internal/genutil/k8s.go

```diff
@@ -21,15 +21,14 @@ import (
 	"path/filepath"
 	"strings"
 
-	genutil "github.com/operator-framework/operator-sdk/commands/operator-sdk/cmd/generate/internal"
 	"github.com/operator-framework/operator-sdk/internal/util/projutil"
 	"github.com/operator-framework/operator-sdk/pkg/scaffold"
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
-func NewGenerateK8SCmd() *cobra.Command {
+func newGenerateK8SCmd() *cobra.Command {
```

Comment:
We have the functions `newGenerateK8SCmd()` and `k8sFunc()`(line 52) defined here in `cmd/operator-sdk/internal/genutil/k8s.go` and in `cmd/operator-sdk/generate/k8s.go`. I think they're only being used in the latter pkg. Do we need this here?

Reply:
We don't. Those were mistakenly copied over. Good catch.